LiskHQ/lisk-mobile
# Show bookmark label instead of address in transaction list - Closes #473

---

### 🚨It'd be better to merge https://github.com/LiskHQ/lisk-mobile/pull/466 first

---

# What was the bug or feature?
#473 

### How did I fix it?
- Subscribed to followedAccounts on accounts reducer
- Passed followedAccounts array to transaction list and used on the render part of transaction list item

## Type of change
- Enhancement (a non-breaking change which adds functionality)

### How to test it?
Open transaction list and check the appearance of a transaction made between you and one of your followed accounts.

# Checklist:
- My code follows the style guidelines of this project
- I have performed a self-review of my own code
- I have commented my code, particularly in hard-to-understand areas
- I have made corresponding changes to the documentation
- I have added tests that prove my fix is effective or that my feature works
- New and existing unit tests pass locally with my changes
